### PR TITLE
Refactor FXIOS-7656 [v119] Fix constraint crash in SyncNow settings cell

### PR DESCRIPTION
**Pull request by lmarceau**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7656)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/17076)

## :bulb: Description
- Fix constraint crash in SyncNow settings cell
- Removed SnapKit usage at the same time

### Before
Was crashing showing this, we were changing constraints in a way that I think is not permitted anymore? (adding a view in the cell, then changing the textLabel UITableViewCell position)
image

### After
Screenshot 2023-10-31 at 3 35 01 PM

## :pencil: Checklist
You have to check all boxes before merging
- [X] Filled in the above information (tickets numbers and description of your work)
- [X] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client/Frontend/Settings/Main/Account/SyncNowSetting.swift [hunk 1]
```diff
@@ -145,26 +145,29 @@ class SyncNowSetting: WithAccountSetting {
...
+        stackView.alignment = .center
+        stackView.distribution = .fillProportionally
+        stackView.spacing = 4
+    }
 
     private lazy var warningIcon: UIImageView = {
-        let imageView = UIImageView(image: UIImage(named: "AmberCaution"))
-        imageView.sizeToFit()
-        return imageView
+        return UIImageView(image: UIImage(named: "AmberCaution"))
```
Comment (adudenamedruby) [inline Client/Frontend/Settings/Main/Account/SyncNowSetting.swift:166]: Should these names be in `ImageIdentifiers`?

Comment (lmarceau) [reply Client/Frontend/Settings/Main/Account/SyncNowSetting.swift:166]: Yes probably 👀 Will do that

**Comment by clarmso** [comment]
@lmarceau @adudenamedruby Is #16960 a duplicate? 😄

**Comment by lmarceau** [comment]
> @lmarceau @adudenamedruby Is #16960 a duplicate? 😄

Yes it seems so 🙏

**Comment by lmarceau** [comment]
@Mergifyio backport release/v119 release/v120

**Pull request merged**